### DISCUSSION TITLE
câu hỏi về Treebanks

### DISCUSSION THREAD

**Discussion by NguyenHuyCung** [undertheseanlp/underthesea#674] [General]
làm sao để import được treebanks ạ, em đang học xử lý ngôn ngữ tự nhiên

**Comment by rain1024** [comment]
@NguyenHuyCung Em muốn dùng treebank nào? Em muốn sử dụng Treebank vào mục đích gì nhỉ?

**Reply by NguyenHuyCung** [reply]
em đang học xử lý ngôn ngữ tự nhiên, em làm bài tập lớn thôi ạ, em muốn dùng treebanks để xây dựng cú pháp, anh có thể giúp em được không ạ .

**Reply by rain1024** [reply]
Em muốn hỏi về VietTreeBank đúng không? https://ieeexplore.ieee.org/document/6719884

VietTreeBank đang sở hữu bởi nhóm VLSP, anh không có quyền chia sẻ dữ liệu này. Em liên hệ với nhóm tác giả VLSP để xin dữ liệu nhé https://vlsp.hpda.vn/demo/?page=resources

**Reply by NguyenHuyCung** [reply]
vâng ạ, em cảm ơn ạ
